public/New-DbaAvailabilityGroup.ps1

```diff
@@ -56,6 +56,14 @@ function New-DbaAvailabilityGroup {
     .PARAMETER Name
         The name of the Availability Group.
 
+    .PARAMETER IsContained
+        Builds the Availability Group as contained. Only supported in SQL Server 2022 or higher.
+
+        https://learn.microsoft.com/en-us/sql/database-engine/availability-groups/windows/contained-availability-groups-overview?view=sql-server-ver16
+
+    .PARAMETER ReuseSystemDatabases
+        Used when rebuilding a cluster where system databases already exist for the contained availability group.
```

Comment:
I would not use the term "cluster" here. Quote from the documentation: "This option is only valid for contained AGs, and specifies that the newly created AG should reuse existing contained system databases for a previous contained AG of the same name. For example, if you had a contained AG with the name MyContainedAG, and wanted to drop and recreate it, you could use this option to reuse the contents of the original contained system databases."